packages/wrangler/src/versions/index.ts

```diff
@@ -275,6 +277,15 @@ export async function versionsUploadHandler(
 		tag: args.tag,
 		message: args.message,
 	});
+
+	if (versionId && name) {
```

Comment:
Why does this need `&& name` but the same code in deploy/deploy.ts doesn't? Not sure which is correct but I think they should be the same, right?

Reply:
This is because the check for whether the name is defined or not (causing an error to be thrown) is deep inside the `versionsUpload()` function. So it should always be defined by the time it gets here. But I felt it was too large a refactor to pull that out.

Reply:
Isn't that the same for deploy? Both call `getScriptName`. Both should be consistent.